davealex/quick-credit
# #165587983 Design Admin Dashboard page

#### What does this PR do?
This PR adds an admin_dashboard.html page to the Project
#### Description of Task to be completed?
A page that gives an admin a high-level overview of crucial platform activities presented in the form of stats - on Users, Applications, Loans, and Repayments 
#### How should this be manually tested?
Navigate to the admin_dashboard.html page and ensure the grids stack properly on mobile devices
#### Any background context you want to provide?
Admin should be presented with quick links to recurrent administrative operations involving: Users, Applications, Loans, and Repayments to improve admin UX
#### What are the relevant pivotal tracker stories?
#165587983
#### Screenshots (if appropriate)
#### Questions:
